frontend/src/App.tsx

```diff
@@ -48,13 +49,38 @@ import { getDownloadURL, ref } from 'firebase/storage'
 import axios, { AxiosResponse } from 'axios'
 import { CourseProvider, StudentProvider } from '@context'
 import { TemplateProvider } from '@context/TemplateContext'
+import React from 'react'
 
 const App = () => {
   const [currentUser, setCurrentUser] = useState<User | null>(null)
   const [authState, setAuthState] = useState<AuthState>('loading')
   const axiosAuthInterceptor = useRef<number | null>(null)
   const [networkError, setNetworkError] = useState<string | null>(null)
 
+  const [needsRefresh, setNeedsRefresh] = useState(false)
+  const checkRefreshDuration = 1000
+  // 600000 // (10 min)
+
+  const handleClose = (
+    event?: React.SyntheticEvent | Event,
+    reason?: string
+  ) => {
+    if (reason === 'clickaway') {
+      return
+    }
+  }
```

Comment:
I don't think you need this function anymore, to do nothing simply don't pass Snackbar the `onClose`prop